sp800-63a/sec9_usability.md

```diff
@@ -70,7 +70,7 @@ In order to ensure users are equipped to make informed decisions about whether t
   * An option to set an appointment for in-person or in-person over remote channels identity proofing sessions to minimize wait times. If walk-ins are allowed, make it clear to users that their wait times may be greater without an appointment.
     * Provide clear instructions regarding setting up an enrollment session appointment and reminders, and how to reschedule existing appointments.
     * Offer appointment reminders and allow users to specify their preferred appointment reminder format(s) (e.g., postal mail, voicemail, email, text message). Users need information such as date, time, location, and a description of required identity evidence.
-* Information on the allowed and required identity evidence and attributes, whether each piece is voluntary or mandatory, and the consequences for not providing the complete set of identity evidence. Users need to know the specific combinations of identity evidence, including requirements specific to a piece of identity evidence (for example, a raised seal on a birth certificate or an original authenticator for obtaining a derived authenticator). This is especially important due to potential difficulties procuring the necessary identity evidence. 
+* Information on the allowed and required identity evidence and attributes, whether each piece is voluntary or mandatory, and the consequences for not providing the complete set of identity evidence. Users need to know the specific combinations of identity evidence, including requirements specific to a piece of identity evidence (for example, a raised seal on a birth certificate or an original authenticator for obtaining a derived credential). This is especially important due to potential difficulties procuring the necessary identity evidence. 
```

Comment:
the original example was derived cred., but after your comment and re-reading I will pull that altogether.